packages/eslint-config-airbnb-base/rules/style.js

```diff
@@ -3,531 +3,823 @@ module.exports = {
-        body: 1
-      },
-      CallExpression: {
-        arguments: 1
+    // DISABLE: Manage via Prettier configuration: {
+    //   tabWidth: 2,
+    //   useTabs: true
+    // }
+    indent: [
+      'error',
+      2,
+      {
+        SwitchCase: 1,
+        VariableDeclarator: 1,
+        outerIIFEBody: 1,
+        // MemberExpression: null,
+        FunctionDeclaration: {
+          parameters: 1,
+          body: 1,
+        },
+        FunctionExpression: {
+          parameters: 1,
+          body: 1,
+        },
+        CallExpression: {
+          arguments: 1,
+        },
+        ArrayExpression: 1,
+        ObjectExpression: 1,
+        ImportDeclaration: 1,
+        flatTernaryExpressions: false,
+        // list derived from https://github.com/benjamn/ast-types/blob/HEAD/def/jsx.js
+        ignoredNodes: [
+          'JSXElement',
+          'JSXElement > *',
+          'JSXAttribute',
+          'JSXIdentifier',
+          'JSXNamespacedName',
+          'JSXMemberExpression',
+          'JSXSpreadAttribute',
+          'JSXExpressionContainer',
+          'JSXOpeningElement',
+          'JSXClosingElement',
+          'JSXFragment',
+          'JSXOpeningFragment',
+          'JSXClosingFragment',
+          'JSXText',
+          'JSXEmptyExpression',
+          'JSXSpreadChild',
+        ],
+        ignoreComments: false,
       },
-      ArrayExpression: 1,
-      ObjectExpression: 1,
-      ImportDeclaration: 1,
-      flatTernaryExpressions: false,
-      // list derived from https://github.com/benjamn/ast-types/blob/HEAD/def/jsx.js
-      ignoredNodes: ['JSXElement', 'JSXElement > *', 'JSXAttribute', 'JSXIdentifier', 'JSXNamespacedName', 'JSXMemberExpression', 'JSXSpreadAttribute', 'JSXExpressionContainer', 'JSXOpeningElement', 'JSXClosingElement', 'JSXFragment', 'JSXOpeningFragment', 'JSXClosingFragment', 'JSXText', 'JSXEmptyExpression', 'JSXSpreadChild'],
-      ignoreComments: false
-    }],
+    ],
 
     // specify whether double or single quotes should be used in JSX attributes
     // https://eslint.org/docs/rules/jsx-quotes
+    // KEEP, don't care about jsx
     'jsx-quotes': ['off', 'prefer-double'],
 
     // enforces spacing between keys and values in object literal properties
+    // SITUATIONAL ANALYSIS: Personally I prefer this, I think it makes the code more readable, but this is entirely subjective
+    // To break this rule down, it means that
+    //
+    // const bleh = {
+    //   someProp: 1
+    //   someOtherProp: 2
+    // }
+    //
+    // would become
+    //
+    // const bleh = {
+    //   someProp:      1
+    //   someOtherProp: 2
+    // }
+    //
+    // or even
+    //
+    // const bleh = {
+    //   someProp      : 1
+    //   someOtherProp : 2
+    // }
+    //
+    // I'm personally a fan of the second option, but this should probably be discussed by the group to see what everyone's preferences are
     'key-spacing': ['error', { beforeColon: false, afterColon: true }],
```

Comment:
SITUATIONAL ANALYSIS: Personally I prefer this, I think it makes the code more readable, but this is entirely subjective.
To break this rule down, it means that:
```
const bleh = {
  someProp: 1,
  someOtherProp: 2
}
```
would become
```
const bleh = {
  someProp:      1
  someOtherProp: 2
}
```
or even
```
const bleh = {
  someProp      : 1,
  someOtherProp : 2
}
```
I'm personally a fan of the second option, but this should probably be discussed by the group to see what everyone's preferences are